controller.Jenkinsfile

```
@@ -0,0 +1,78 @@
+#!/usr/bin/env groovy
+
+def SOURCE_REPO = 'devfile/devworkspace-operator' //source repo from which to find and sync commits to pkgs.devel repo
+def DWNSTM_REPO = 'containers/web-terminal' // dist-git repo to use as target for everything
+
+def SOURCE_BRANCH = 'master'         // target branch in GH repo, eg., master or v1.0.x
```

I wonder if we should create branch for web-terminal 1.0 in the devworkspace controller, like v1alpha1...

---

[v1.0.0-alphax](https://github.com/devfile/devworkspace-operator/tree/v1.0.0-alphax) is created

```suggestion
def SOURCE_BRANCH = 'master'         // target branch in GH repo, eg., master or v1.0.0-alphax
```